### PR TITLE
Updating for Zcash default ports

### PR DESCRIPTION
**Pull request by garethtdavies**
I haven't checked yet to see if all of these RPC methods are supported but this fixes up default ports for Bitcoin->Zcash

### CODE REVIEW

**Review by dconnolly — APPROVED** [review]
Thanks for the catch! 😁

File: JSON-RPC.md [hunk 1]
```diff
@@ -1,122 +1,122 @@
 # JSON-RPC
 
-The JSON-RPC interface is served on port :8332 for mainnet and :18332 for testnet unless you specified otherwise. So if you are using testnet, you will need to change the port in the sample curl requests shown below.
+The JSON-RPC interface is served on port :8232 for mainnet and :18232 for testnet unless you specified otherwise. So if you are using testnet, you will need to change the port in the sample curl requests shown below.
 
 ### Network
 
-The Parity Zcash `network` interface.
+The Zebra Zcash `network` interface.
```
Comment (dconnolly) [inline JSON-RPC.md]:
```suggestion
The Zebra `network` interface.
```

File: JSON-RPC.md [hunk 2]
```diff
@@ -1,122 +1,122 @@
...
 
 Get the peer count.
 
-    curl -H 'content-type: application/json' --data-binary '{"jsonrpc": "2.0", "id":"1", "method": "getconnectioncount", "params": [] }' localhost:8332
+    curl -H 'content-type: application/json' --data-binary '{"jsonrpc": "2.0", "id":"1", "method": "getconnectioncount", "params": [] }' localhost:8232
 
 ### Blockchain
 
-The Parity Zcash `blockchain` data interface.
+The Zebra Zcash `blockchain` data interface.
```
Comment (dconnolly) [inline JSON-RPC.md]:
```suggestion
The Zebra `blockchain` data interface.
```

File: JSON-RPC.md [hunk 3]
```diff
@@ -1,122 +1,122 @@
...
 
 Get statistics about the unspent transaction output set.
 
-    curl -H 'content-type: application/json' --data-binary '{"jsonrpc": "2.0", "method": "gettxoutsetinfo", "params": [], "id":1 }' localhost:8332
+    curl -H 'content-type: application/json' --data-binary '{"jsonrpc": "2.0", "method": "gettxoutsetinfo", "params": [], "id":1 }' localhost:8232
 
 ### Miner
 
-The Parity Zcash `miner` data interface.
+The Zebra Zcash `miner` data interface.
```
Comment (dconnolly) [inline JSON-RPC.md]:
```suggestion
The Zebra `miner` data interface.
```

File: JSON-RPC.md [hunk 4]
```diff
@@ -1,122 +1,122 @@
...
 
 Get block template for mining.
 
-    curl -H 'content-type: application/json' --data-binary '{"jsonrpc": "2.0", "method": "getblocktemplate", "params": [{"capabilities": ["coinbasetxn", "workid", "coinbase/append"]}], "id":1 }' localhost:8332
+    curl -H 'content-type: application/json' --data-binary '{"jsonrpc": "2.0", "method": "getblocktemplate", "params": [{"capabilities": ["coinbasetxn", "workid", "coinbase/append"]}], "id":1 }' localhost:8232
 
 ### Raw
 
-The Parity Zcash `raw` data interface.
+The Zebra Zcash `raw` data interface.
```
Comment (dconnolly) [inline JSON-RPC.md]:
```suggestion
The Zebra `raw` data interface.
```

**Review by dconnolly — CHANGES_REQUESTED** [review]
I added some nomenclature suggested changes, if you can apply those please.

**Comment by garethtdavies** [comment]
Addressed those comments and force pushed to combine to a single commit.

**Review by dconnolly — APPROVED** [review]
🙏

**Pull request merged**